WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -205,7 +205,7 @@ def post_announcement
       full_post = nil
       ActiveRecord::Base.transaction do
         full_post = create_post_and_redirect(title: title, body: body, author: current_user, tags: "competitions,new", world_readable: true)
-        comp.update!(announced_at: Time.now)
+        comp.update!(announced_at: Time.now, announced_by: current_user.id)
```

Comment:
It would be nice to motivate this change with a corresponding change to spec/controllers/competitions_controller_spec.rb. It should be a pretty straightforward change to one of the existing tests.

Reply:
Ok, I'm modifying one of the existing tests to add this.